### DISCUSSION TITLE
Valve Spam

### DISCUSSION THREAD

**Discussion by TheFruxz** [community/community#143644] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Heya,
since some days my feed is suddenly filled with random people starring valve repos, this suddenly appeared to be the case, although I changed nothing in the last days.
It wouldnt bother me, if this is not the case every 30 minutes.

<img width="925" alt="image" src="https://github.com/user-attachments/assets/ceefb874-ff44-4328-8dfd-f49d52c86fc9">

**Comment by TheFruxz** [comment]
In the menu, it tells me that I would follow everone of these persons, but I don't.
<img width="903" alt="image" src="https://github.com/user-attachments/assets/a1697be8-2026-41a5-b2a9-1c6c9342a576">

It looks like a bug, where everyone is now referred as a follower, ig?

**Comment by cpz** [comment]
Exactly same happens to me
![image](https://github.com/user-attachments/assets/a8af20c5-adf2-4e3c-b4b5-52a52aedc437)

**Comment by RVxLab** [comment]
Getting the same with non-valve repositories and users my feed says I sponsor, who I certainly don't. I am however sponsoring the account/org in the screenshot.

Looks to be a bug.

![image](https://github.com/user-attachments/assets/2b5e27ce-3fc5-4bc8-bb05-cb45631e57bd)

**Comment by sefinek** [comment]
This is definitely a bug. I'm experiencing the same issue. However, in my case, it only occurs with Express.js repositories because I follow that organization
![{5609BF9B-D169-4F2A-8183-005C746F0794}](https://github.com/user-attachments/assets/050a6c78-9522-409d-828b-8ed25a2f16c3)

**Comment by LiteBrite82** [comment, marked as answer]
Hi Community!

Thank you all for your posts and comments on the recent Feed bug. The bug was related to starred repositories, which is why we see some people saying the fix is to uncheck "Include events from starred repositories".

The Feed team has pushed an update to address the issue; [please leave us feedback in this separate post](https://github.com/orgs/community/discussions/143676) so we can monitor its effectiveness.
